packages/vats/ava.bench.config.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  files: ['test/**/bench-*.js'],
+  concurrency: 2,
```

Comment:
why 2? 

if you omit this config option it will default to number of CPU cores. 
https://github.com/avajs/ava/blob/main/docs/06-configuration.md#options

Reply:
I was just blindly replicating the `"ava"` clause from `package.json`.  I have no position on the actual values of these parameters.

Reply:
I think it should be omitted to use the default but let's ask Chesterton.

@michaelfig why does `vats` have this defined? https://github.com/Agoric/agoric-sdk/pull/7586/commits/884fa3b3eeb9b74730c961229ac4392ba045f14e

Reply:
> why does `vats` have this defined? [884fa3b](https://github.com/Agoric/agoric-sdk/commit/884fa3b3eeb9b74730c961229ac4392ba045f14e)

I think this is a relic from before the bundling process was actually locking the bundle cache entries (if you recall, I had forgotten to replace the SwingSet bundler with the fixed Endo version).  Now, I think there's no longer an issue.

Reply:
Oh good. https://github.com/Agoric/agoric-sdk/pull/7974

Reply:
I echo @turadg's comment, if for no other reason than I was persuaded by his argument for letting the `concurrency` option default and the landed version of this PR has it thus.  So this means I don't have to change it back :-)